tasmota/tasmota_xsns_sensor/xsns_102_ld2402.ino

```diff
@@ -836,7 +813,7 @@ void CmndLd2402Mode(void) {
   ArgV(Argument,1);
   memset(LD2402.cmnd_param, 0x00, 6);
   LD2402.cmnd_param[2] = atoi(Argument) ? 0x04 : 0x64;
-  Response_P(PSTR(D_COMMAND_PREFIX_JSON "%d}"), D_CMD_MODE, (0x04 == LD2402.cmnd_param[2]));
+  Response_P(PSTR(D_COMMAND_PREFIX_JSON "\"%s\"}"), D_CMD_MODE, (0x04 == LD2402.cmnd_param[2] ? "Engineering" : "Normal"));
```

Comment:
Every response needs a valid language translation so keeping the number is much more code friendly than adding text messages.

Reply:
My error. It's OK.